wapopartners/arctrack
# Fix (potentially disastrous) commit overwriting mistake

## On this branch

_fix scary regression_

## To test

* pull the Arctrack branch and run `yarn test`

* In your `tronc` repo, [fetch this branch](https://github.com/wapopartners/Tronc-PageBuilder-Features/tree/arctrack-regression-fix) `arctrack-regression-fix`.

* run `yarn` && `yarn build`

* Open `http://localhost/pb/entertainment/movies/la-ca-mn-sneaks-strong-women-summer-films-20170421-story.html`

* check the console for the following:

on page load:
<img width="1261" alt="screen shot 2018-04-02 at 6 09 57 pm" src="https://user-images.githubusercontent.com/8948627/38218858-280647be-36a1-11e8-94db-e4324f468052.png">

scroll over an inline photo:
<img width="1257" alt="screen shot 2018-04-02 at 6 11 37 pm" src="https://user-images.githubusercontent.com/8948627/38218901-52c19422-36a1-11e8-979e-710f2e4bfe08.png">

scroll over the video embed:
<img width="1249" alt="screen shot 2018-04-02 at 6 12 09 pm" src="https://user-images.githubusercontent.com/8948627/38218917-682ec370-36a1-11e8-86a3-430a06dd055f.png">

open the lightbox:
<img width="1246" alt="screen shot 2018-04-02 at 6 12 46 pm" src="https://user-images.githubusercontent.com/8948627/38218932-808d2bf0-36a1-11e8-8051-ab4498880fb4.png">

click to share something:
<img width="1243" alt="screen shot 2018-04-02 at 6 13 38 pm" src="https://user-images.githubusercontent.com/8948627/38218949-98b2a41c-36a1-11e8-85c2-baee7fcc6e63.png">

click search/ click the topics tray
<img width="862" alt="screen shot 2018-04-02 at 6 14 08 pm" src="https://user-images.githubusercontent.com/8948627/38218975-afb60c44-36a1-11e8-869b-e3d8b29ab488.png">




## QA

#### OS

#### Browsers (last two versions)
- [ ] IE 11 and Edge

#### Devices
- [ ] Galaxy
- [ ] iPhone 6s
- [ ] etc.

## Comments
